components/sup/src/templating/helpers/str_concat.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2017 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use handlebars::{Handlebars, Helper, HelperDef, RenderContext};
+
+use super::super::RenderResult;
+
+#[derive(Clone, Copy)]
+pub struct StrConcatHelper;
+
+impl HelperDef for StrConcatHelper {
+    fn call(&self, h: &Helper, _: &Handlebars, rc: &mut RenderContext) -> RenderResult<()> {
+        let list: Vec<String> = h.params()
+            .iter()
+            .map(|v| v.value())
```

Comment:
You're looking for [filter_map](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.filter_map) but this is totally fine :)